src/services/domain/createTeam.ts

```diff
@@ -0,0 +1,7 @@
+import { Team } from "../../pages/CreateTeamPage/CreateTeamPage";
```

Comment:
this file **createTeam.ts** was already deleted since it is declared in the TeamService. Probably it stayed here from an old merge